src/frontend/vite.config.ts

```diff
@@ -1,10 +1,18 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig, loadEnv } from 'vite';
+import react from '@vitejs/plugin-react';
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react()],
-  build: {
-    manifest: true,
-  },
-})
+export default defineConfig(({ mode }) => {
+  // import.meta.env.VITE_PORT available here with: process.env.VITE_PORT
+  process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
```

Comment:
is this function only used at the time of bundling? @binggaofw

Reply:
to my understanding , yes it should only used at build the bundle